torchtune/models/qwen2_5/_model_builders.py

```diff
@@ -0,0 +1,629 @@
+    from https://huggingface.co/Qwen/Qwen2.5-32B-Instruct
+
+    Returns:
+        TransformerDecoder: Instantiation of Qwen2.5 32B model
+    """
+    return qwen2(
+        vocab_size=152064,
+        num_layers=64,
+        num_heads=40,
+        num_kv_heads=8,
+        embed_dim=5120,
+        intermediate_dim=27648,
+        max_seq_len=32768,
+        attn_dropout=0.0,
+        norm_eps=1e-06,
+        rope_base=1000000.0,
+    )
+
+
+def qwen2_72b() -> TransformerDecoder:
+    """
+    Builder for creating a Qwen2.5 model initialized w/ the default 72B parameter values
+    from https://huggingface.co/Qwen/Qwen2.5-72B-Instruct
+
+    Returns:
+        TransformerDecoder: Instantiation of Qwen2.5 72B model
+    """
+    return qwen2(
+        vocab_size=152064,
+        num_layers=80,
+        num_heads=64,
+        num_kv_heads=8,
+        embed_dim=8192,
+        intermediate_dim=29568,
+        max_seq_len=32768,
+        attn_dropout=0.0,
+        norm_eps=1e-06,
+        rope_base=1000000.0,
+    )
+
+
+def qwen2_tokenizer(
+    path: str,
+    merges_file: str,
+    special_tokens_path: Optional[str] = None,
+    max_seq_len: Optional[int] = None,
+    prompt_template: Optional[
+        str
+    ] = "torchtune.models.qwen2_5._prompt_template.Qwen2_5ChatTemplate",
+    **kwargs,
+) -> Qwen2Tokenizer:
+    """
+    Tokenizer for Qwen2.5.
+
+    Args:
+        path (str): path to the vocab.json file.
+        merges_file (str): path to the merges.txt file.
+        special_tokens_path (Optional[str]): Path to ``tokenizer.json`` from Hugging Face
+            model files that contains all registered special tokens, or a local json file
+            structured similarly. Default is None to use the canonical Qwen2.5 special tokens.
+        max_seq_len (Optional[int]): A max sequence length to truncate tokens to.
+            Default: None
+        prompt_template (Optional[_TemplateType]): optional specified prompt template.
+            If a string, it is assumed to be the dotpath of a :class:`~torchtune.data.PromptTemplateInterface`
+            class. If a dictionary, it is assumed to be a custom prompt template mapping role to the
+            prepend/append tags.
+            Default is :class:`~torchtune.models.qwen2_5._prompt_template.Qwen2_5ChatTemplate`.
+
+    Returns:
+        Qwen2Tokenizer: Instantiation of the Qwen2.5 tokenizer
+    """
+    special_tokens = (
+        QWEN2_5_SPECIAL_TOKENS
+        if special_tokens_path is None
+        else parse_hf_tokenizer_json(special_tokens_path)
+    )
+
+    if prompt_template is not None:
+        prompt_template = _get_prompt_template(prompt_template)
+
+    return Qwen2Tokenizer(
+        path=path,
+        merges_file=merges_file,
+        special_tokens=special_tokens,
+        max_seq_len=max_seq_len,
+        prompt_template=prompt_template,
+        **kwargs,
+    )
+
+
+def lora_qwen2_7b(
```

Comment:
this is a nit, but why are the model sizes out of order down here?

Reply:
(also just make sure that any changes you wind up making to the non-lora builders' params get propagated down here as well)